desktop/components/jump/react.js

```diff
@@ -21,7 +21,7 @@ export default class Jump extends Component {
     const { threshold, direction, element, offset } = this.props
 
     this.backboneView = new JumpView({
-      threshold: threshold || this.$(window).height(),
+      threshold: threshold || this.$(window).height() + 500,
```

Comment:
Can we set this from where we use it in the `/auction` page? Otherwise it'll be the default for all future users of this component.